generator/tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
   "compilerOptions": {
     "types": ["node"],
-    "moduleResolution": "node",
+    "module": "NodeNext",
+    "isolatedModules": true,
     "checkJs": true,
     "allowJs": true,
+    "noEmit": true,
     "strict": true,
-    "resolveJsonModule": true,
```

Comment:
It didn't look like this was needed anymore, and latest ts5.7 makes backcompat a pain, so I removed it.